styles/products.css

```diff
@@ -47,3 +51,23 @@ footer {
   grid-template-columns: repeat(auto-fit, minmax(25rem, 1fr));
   grid-gap: 2rem;
 }
+
+@media (max-width: 820px) {
+  .wrapper {
+    grid-template-columns: 1fr;
+    grid-template-areas:
+      "header header"
+      "hamburger-menu hamburger-menu"
+      "filters filters"
+      "products products"
+      "footer footer";
+  }
+  #filters{
```

Comment:
is there any specific reason for id selector here? id selector is only used for very unique elements as per my knowledge.

Reply:
filters section is unique on products page that's why I used id